bin/node/src/chain_spec.rs

```diff
@@ -129,37 +118,49 @@ pub struct ChainParams {
     #[structopt(long, default_value = "account_secret")]
     pub account_key_file: String,
 
+    // The length of a session
```

Comment:
can we provide a default for these two, seems like session_period() is exactly that

Reply:
done